examples/post_training_quantization/torch/mobilenet_v2/main.py

```diff
@@ -150,27 +151,31 @@ def transform_fn(data_item):
 
 dummy_input = torch.randn(1, 3, 224, 224)
 
-fp32_onnx_path = f"{ROOT}/mobilenet_v2_fp32.onnx"
-torch.onnx.export(
-    torch_model.cpu(),
-    dummy_input,
-    fp32_onnx_path,
-    input_names=["input"],
-    output_names=["output"],
-    dynamic_axes={"input": {0: "-1"}},
-)
-ov_model = mo.convert_model(fp32_onnx_path)
-
-int8_onnx_path = f"{ROOT}/mobilenet_v2_int8.onnx"
-torch.onnx.export(
-    torch_quantized_model.cpu(),
-    dummy_input,
-    int8_onnx_path,
-    input_names=["input"],
-    output_names=["output"],
-    dynamic_axes={"input": {0: "-1"}},
-)
-ov_quantized_model = mo.convert_model(int8_onnx_path)
+if USE_EXPERIMENTAL_MO_CONVERTOR:
```

Comment:
I would recommend switching to mo.convert_model(...) when a public OV wheel is available with the necessary changes.

Reply:
removed